Formula/tomsfastmath.rb

```diff
@@ -0,0 +1,31 @@
+class Tomsfastmath < Formula
+  desc "Fast, large integer arithmetic library written in C"
+  homepage "http://www.libtom.net"
+  url "https://github.com/libtom/tomsfastmath/releases/download/v0.13.0/tfm-0.13.tar.bz2"
+  sha256 "8defe6f20759a455ade46b2f3c4de46d2358a6222236f78012736a6aeed63407"
+
+  def install
+    system "make"
+
+    include.install "src/headers/tfm.h"
```

Comment:
Is there no `make install`? If not, please file an issue to https://github.com/libtom/tomsfastmath or, ideally, a pull request to add one. Thanks!